lib/erb_lint/linters/deprecated_classes.rb

```diff
@@ -79,20 +78,18 @@ module StartTagHelper
 
       # attributes can be empty or have an attribute value
       ATTRIBUTE_PATTERN = %r{
-        #{ATTRIBUTE_NAME_PATTERN}        # attribute name
+        [^\s"'>\/=]+                     # attribute name
         (
           \s*=\s*                        # any whitespace around equals sign
           (#{ATTRIBUTE_VALUE_PATTERN})   # attribute value
         )?                               # attributes can be empty or have an assignemnt.
       }x
 
-      # Start tag Patterns
+      # Start tag Pattern
       # https://www.w3.org/TR/html5/syntax.html#syntax-start-tag
 
-      TAG_NAME_PATTERN = /[A-Za-z0-9]+/ # maybe add _ < ? etc later since it gets interpreted by some browsers
-
       START_TAG_PATTERN = %r{
-        <(#{TAG_NAME_PATTERN})         # start of tag with tag name
+        <([A-Za-z0-9]+)                # tag name
```

Comment:
![image](https://cloud.githubusercontent.com/assets/2578036/15780526/2bf4ff4a-2970-11e6-980b-acb0812c17ae.png)

According to the HTML5 Spec, yes.